extensions/amp-consent/0.1/consent-ui.js

```diff
@@ -354,6 +363,7 @@ export class ConsentUI {
   handleReady_(data) {
     this.initialHeight_ = DEFAULT_INITIAL_HEIGHT;
     this.enableBorder_ = DEFAULT_ENABLE_BORDER;
+    this.lightboxEnabled_ = false;
```

Comment:
Why reset as `false`?

Reply:
Potentially, a new iframe is being shown (via re-prompt), so better to reverify all conditions.